### PR TITLE
KCL 2.2.2 release

### PR DESCRIPTION
**Pull request by ashwing**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The recommended way to use the KCL for Java is to consume it from Maven.
   <dependency>
       <groupId>software.amazon.kinesis</groupId>
       <artifactId>amazon-kinesis-client</artifactId>
-      <version>2.2.1</version>
+      <version>2.2.2</version>
```
Comment (micah-jaffe) [inline README.md]: Minor - while we're here can we also update the 1.x version in the master README to be the most recent 1.11.2?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,21 @@
 # Changelog
+
+### Release 2.2.2 (August 19, 2019)
+
+[Milestone#36](https://github.com/awslabs/amazon-kinesis-client/milestone/36)
+* Fix to prevent invalid ShardConsumer state transitions due to rejected executor service executions
+  * [PR#560](https://github.com/awslabs/amazon-kinesis-client/pull/560)
+* Fixing bug where initial subscription failure causes shard consumer to get stuck
+  * [PR#562](https://github.com/awslabs/amazon-kinesis-client/pull/562)
+* Making CW publish failures visible
```
Comment (micah-jaffe) [inline CHANGELOG.md]: We want the changelog text to be understandable by people without context on the specific changes. So usually we'll change the wording of the PR commits a little - e.g. this line would be something like "Make CloudWatch publishing failures visible by adding additional logging in <className>"

Comment (ashwing) [reply CHANGELOG.md]: addressing

File: README.md (hunk 1 above)
Comment (ashwing) [reply README.md]: addressing

**Pull request merged**